plugins/file-open-save-new/core/src/main/java/org/pentaho/di/plugins/fileopensave/dialog/FileOpenSaveDialog.java

```diff
@@ -652,6 +652,12 @@ private String[] getStringsAtEachDirectory( String targetPath, char pathSplitter
         }
       }
     } else {
+      if ( targetPath.startsWith( "file:///" ) ) {
+        targetPath = targetPath.substring( 7 );
```

Comment:
Can you please add a comment about what "7" mean?

Reply:
done.